automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
Just figured out you are using dict but you are extracting params using the function. So if you change the return to tuple you will have params with no need to key look up:

```python
return get_attribute_value(data, ak_name, 'content view'), get_attribute_value(data, ak_name, 'lifecycle environment')
```
You can modify the code using function from 

```python
cv_lenv = hammer_determine_cv_and_env_from_ak(activation_key, '1')
cv_name, env_name = cv_lenv['cv'], cv_lenv['lenv']
```
to
```python
cv_name, env_name = hammer_determine_cv_and_env_from_ak(activation_key, '1')
```